src/contributing/structure.md

```diff
@@ -0,0 +1,21 @@
+# Documentation Structure
+
+The directory structure is meant to feel comfortable for those familiar with the data platform:
+
+```
+.
+|--src
+   |--datasets - contains dataset level documentation
+   |--tools - contains tool level documentation
+   |--concepts - contains tutorials meant to introduce a new concept to the reader
+   |--cookbooks - focused code examples for reference
+```
+
+This documentation is meant to take the reader from beginner to expert.
+
+* Getting Started: Introduce concepts, and get some simple analysis completed so the user understands the amount of work involved / what the product feels like. Primarily intended for people new to Mozilla's data platform.
```

Comment:
Getting Started: Introduces concepts and provides information on how to perform and complete a simple analysis so you can assess the the volume of work that may be required and gain a better a better understanding about the product. This section is intended for anyone who is new to Mozilla's data platform.